### PR TITLE
Adds openshift/image-inspector container

### PR DESCRIPTION
**Pull request by navidshaikh**
Adds build parameters to build openshift/image-inspector container.

### CODE REVIEW

**Comment by navidshaikh** [comment]
#dotests

**Comment by navidshaikh** [comment]
ok to test

**Comment by navidshaikh** [comment]
ok to test

**Comment by navidshaikh** [comment]
@mohammedzee1000 : PTAL at CI console, all the container index validations seems to be passing the tests - I am not sure though why the CI is reporting failure otherwise.

File: index.d/openshift.yml [hunk 1]
```diff
@@ -822,3 +822,15 @@ Projects:
...
         - openshift/base-centos7:latest
+
+ - id: 60
+   app-id: openshift
+   job-id: image-inspector
+   git-url: https://github.com/openshift/image-inspector
+   git-branch: master
+   git-path: /
+   target-file: Dockerfile
+   disred-tag: latest
```
Comment (mohammedzee1000) [inline index.d/openshift.yml]: I think this is desired-tag?

File: index.d/openshift.yml [hunk 2]
```diff
@@ -822,3 +822,15 @@ Projects:
     depends-on:
         - centos/centos:latest
         - openshift/base-centos7:latest
+
+ - id: 60
```
Comment (mohammedzee1000) [inline index.d/openshift.yml]: Alignment is off

**Comment by mohammedzee1000** [comment]
LGTM, merging

**Pull request merged**